onboarding/src/Components/UserListSearch.js

```diff
@@ -1,54 +1,101 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
 
-
-function UserListSearch() {
-    let locations = ["Warszawa", "Łódź", "Poznań", "Gdańsk", "Wrocław"], dataOptions = [];
-    dataOptions = locations.map( (city, i) =>
+function UserListSearch({ users, setSearchResult }) {
+    const [searchName, setSearchName] = useState("");
+    const [searchDepartment, setSearchDepartment] = useState("");
+    const [searchPosition, setSearchPosition] = useState("");
+    const [searchLocation, setSearchLocation] = useState("");
+    const locations = [
+        "Białystok", 
+        "Bydgoszcz", 
+        "Gdańsk", 
+        "Katowice",
+        "Kielce",
+        "Kraków",
+        "Lublin",
+        "Łódź",
+        "Olsztyn",
+        "Opole",
+        "Poznań",
+        "Rzeszów",
+        "Szczecin",
+        "Warszawa",
+        "Wrocław",
+        "Zielona Góra"
+    ];
+    const dataOptions = locations.map( (city, i) =>
```

Comment:
Same here bec. of:
https://github.com/binawork/OnlineOnboarding/pull/149/files#r545352316